examples/shader/post_processing.rs

```diff
@@ -283,7 +283,7 @@ impl FromWorld for PostProcessPipeline {
                     ty: BindingType::Buffer {
                         ty: bevy::render::render_resource::BufferBindingType::Uniform,
                         has_dynamic_offset: false,
-                        min_binding_size: None,
+                        min_binding_size: Some(PostProcessSettings::min_size()),
```

Comment:
I'm not sure you need this, but it certainly doesn't hurt.

Reply:
It is necessary